### PR TITLE
minor refactor to named method in WorkOrdersController

### PR DESCRIPTION
**Pull request by afeld**
Per suggestion in https://github.com/18F/C2/pull/656/files#r42406827.

/cc @jessieay

### CODE REVIEW

File: app/controllers/ncr/work_orders_controller.rb [hunk 1]
```diff
@@ -74,5 +74,9 @@ def add_approvals
         @model_instance.setup_approvals_and_observers(@approver_email)
       end
     end
+
+    def skip_state_update?
```
Comment (afeld) [inline app/controllers/ncr/work_orders_controller.rb]: Very open to suggestions about what to call this – don't love it.

File: app/controllers/ncr/work_orders_controller.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```
Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: still not a fan of unless! can we change this to an `if` ?

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: how about: `if model_changing_and_not_emergency` ? It's long and blah, but it does explain what's happening!

Comment (afeld) [inline app/controllers/ncr/work_orders_controller.rb]: Ha. Well, that does describe (literally) what the method is doing, but I was going for a name that would convey why those things matter. Does that make sense?

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: I guess I am not 100% sure why they matter, which is probably why this is hard for me. Perhaps you can explain? I looked into the controller to see the important for `model_changing` but I am still not clear

Comment (pkarman) [inline app/controllers/ncr/work_orders_controller.rb]: This feels like one of those cases where some apt comments would explain the "why" and the method can just be the "what".

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: Why can't the method name explain the why!?

Comment (pkarman) [inline app/controllers/ncr/work_orders_controller.rb]: I'm really on +0 on `if model_changing_and_not_emergency?`

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: @pkarman suggestions on something you would be +1 on would be helpful!

Comment (pkarman) [inline app/controllers/ncr/work_orders_controller.rb]: how about
```
if subscribers_require_notification
```

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: :heart:

Comment (jessieay) [inline app/controllers/ncr/work_orders_controller.rb]: although I might add a `?` to the end of that to indicate its boolean-ness

**Comment by afeld** [comment]
Updated!

**Comment by jessieay** [comment]
this looks good! 

should we be concerned that this change required no spec changes? I would expect that L146 in spec/controllers/ncr/work_orders_controller_spec.rb would fail now because it deals with updates  and emergency status.

**Comment by afeld** [comment]
> should we be concerned that this change required no spec changes?

No. `setup_approvals_and_observers` checks for `emergency`, and `email_approvers` does nothing, since emergency requests have none. Now this PR is just taking out a redundant check.

**Comment by jessieay** [comment]
Got it, then looks good to :ship: 

How would someone QA this change? Or does it not need QAing?

**Comment by afeld** [comment]
## Testing notes

* [ ] Update an emergency request, without changing the approver
* [ ] Update an emergency request, changing the approver

**Comment by pkarman** [comment]
I will QA this.

**Comment by pkarman** [comment]
Looks fine on staging.

**Pull request merged**